roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -27,6 +38,52 @@ No initialization routine runs for this module.
 
 ## Routines
 
+There are two routines intended for use by the relay chain extrinsics: `ensure_downward_messages_fits`
+and `queue_downward_messages`. The former function is used before performing relay chain operations
```

Comment:
This reads a bit like the `ensure_downward_messages_fits` and `queue_downward_messages` are extrinsics

Reply:
I rewrote this part. I hope now it is more clear.

Reply:
What is meant by "intended for use by the relay chain extrinsics"?

Reply:
(I see this was updated now and was reviewing commit-by-commit)